cli/cmd/component.go

```diff
@@ -1170,15 +1173,23 @@ func downloadProgress(complete chan int8, path string, sizeB int64) {
 				mb := float64(size) / (1 << 20)
 
 				if mb > previous {
-					cli.spinner.Suffix = fmt.Sprintf("%s Downloaded: %.0fmb", spinnerSuffix, mb)
+					if !cli.nonInteractive {
+						cli.spinner.Suffix = fmt.Sprintf("%s Downloaded: %.0fmb", spinnerSuffix, mb)
+					} else {
+						cli.OutputHuman("..Downloaded: %.0fmb\n", mb)
```

Comment:
The `--noninteractive` flag should turn off spinners, progress, prompts, etc. Which means that printing out is not the right thing. 

@dmurray-lacework @jon-stewart Could someone please modify this behavior to use the logger instead of output to standard output?